dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/WorkerGroupServiceImpl.java

```diff
@@ -84,6 +84,7 @@ public class WorkerGroupServiceImpl extends BaseServiceImpl implements WorkerGro
      * @return create or update result code
      */
     @Override
+    @Transactional(rollbackFor = Exception.class)
```

Comment:
Please don't set the rollbackFor, if you set this then this method will only rollback a checked Exception, but you can know this method will not throw a checked Exception.
```suggestion
    @Transactional
```

Reply:
yeah, thx for this, 'If no rules are relevant to the exception, it will be treated like DefaultTransactionAttribute (rolling back on runtime exceptions)', without adding properties such as rollbackFor, Spring will roll back when Unchecked Exceptions are encountered, not only RuntimeException, but also Error. I will optimize it later.

Reply:
Created an improvement [https://github.com/apache/dolphinscheduler/issues/10791] Transaction checking and optimization will be handled in a new pr.

Reply:
Please fix this soon.